internal/go_repository.bzl

```diff
@@ -402,12 +407,21 @@ def _generate_package_info(*, importpath, version):
     # scheme:type/namespace/name@version?qualifiers#subpath
     purl = "pkg:golang/{namespace_and_name}@{version}".format(
         namespace_and_name = importpath,
-        version = version,
-    ) if version else None
+        version = version if version else "unknown",
```

Comment:
Is this a well-known special string? Based on what the spec says, shouldn't we leave this empty and also drop the `@`?

Reply:
You're right, removed